test_build_proofs failing

Observed on Linux (https://jenkins-ci.tribler.org/job/ipv8/job/test_ipv8_PR_unittests_linux/1497/console):

```
test_task_with_exception_ignore (ipv8.test.test_taskmanager.TestTaskManager) ... ERROR:TaskManager:Task resulted in error: division by zero
ok

======================================================================
FAIL: test_build_proofs (ipv8.test.attestation.wallet.irmaexact.test_builder.TestBuilder)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jenkins/.local/lib/python3.7/site-packages/asynctest/case.py", line 297, in run
    self._run_test_method(testMethod)
  File "/home/jenkins/.local/lib/python3.7/site-packages/asynctest/case.py", line 352, in _run_test_method
    result = method()
  File "/home/jenkins/ci/workspace/ipv8/test_ipv8_PR_unittests_linux/ipv8/test/attestation/wallet/irmaexact/test_builder.py", line 85, in test_build_proofs
    self.assertFalse(proof.Verify(self.testPubK, sig, 10, nonce))
AssertionError: True is not false
```

--- a/ipv8/test/attestation/wallet/irmaexact/test_builder.py
+++ b/ipv8/test/attestation/wallet/irmaexact/test_builder.py
@@ -83,5 +83,5 @@ class TestBuilder(unittest.TestCase):
         proof = issuer.proveSignature(sig, nonce)
 
         self.assertTrue(proof.Verify(self.testPubK, sig, context, nonce))
-        self.assertFalse(proof.Verify(self.testPubK, sig, context, 10))
-        self.assertFalse(proof.Verify(self.testPubK, sig, 10, nonce))
+        self.assertFalse(proof.Verify(self.testPubK, sig, context, nonce + 1))
+        self.assertFalse(proof.Verify(self.testPubK, sig, context + 1, nonce))

